ui/lib/core/addon/components/overview-card.hbs

```diff
@@ -10,10 +10,14 @@
   data-test-overview-card-container={{@cardTitle}}
   ...attributes
 >
-  <div class="flex row-wrap space-between has-bottom-margin-s" data-test-overview-card={{@cardTitle}}>
-    <Hds::Text::Display @weight="bold" @size="300" data-test-overview-card-title={{@cardTitle}}>
-      {{@cardTitle}}
-    </Hds::Text::Display>
+  <div class="flex row-wrap space-between" data-test-overview-card={{@cardTitle}}>
```

Comment:
While both `customSubtext` and `@subText` are passable, I opted to make `customTitle` and `@cardTitle` mutually exclusive